scorecard.js

```diff
@@ -0,0 +1,67 @@
+const Frame = require("./frame");
+// const readline = require("readline");
+// const rl = readline.createInterface({
+//   input: process.stdin,
+//   output: process.stdout,
+// });
+
+class Scorecard {
+  constructor() {
+    this.frames = [];
+    this.bonusRolls = 0
+    this.bonus = 0
+  }
+
+  getFrames() {
+    return this.frames;
+  }
+
+  playRound(roll1, roll2) {
+    const frame = new Frame();
+    frame.setRoll1(roll1);
+    frame.setRoll2(roll2);
+    frame.addToTotal(frame.getRoll1() + frame.getRoll2());
+    return frame;
+  }
+
+  addFrame(frame) {
+    this.frames.push(frame);
+  }
+
+  updateFrameTotal(frameNum, score) {
+    this.frames[frameNum - 1].addToTotal(score)
```

Comment:
**Very minor** formatting thing - looks like you started off with semi-colons consistently, but dropped off from this line